spec/db/models.rb

```diff
@@ -95,6 +99,11 @@ def book_with_most_bookmarks
   virtual_attribute :first_book_author_name, :string, :uses => {:books => :author_name}
   # uses another virtual attribute that uses a relation
   virtual_attribute :upper_first_book_author_name, :string, :uses => :first_book_author_name
+  # uses another virtual_attribute that uses a complex relation
+  # Please do not "fix" syntax of the uses here
+  #   actually testing this format with an array: [:bwmb, {:books => co_a}]
+  #   vs "better" format not helping us with tests: {:bwmb => {}, :books => co_a}
```

Comment:
The wording here is strange to me - I think this is clearer and more direct

```suggestion
  # NOTE: Please do not change the :uses format here.
  #   This intentionally tests :uses with an array: [:bwmb, {:books => co_a}]
  #   vs a "better" format such as: {:bwmb => {}, :books => co_a}
```